testing/src/main/java/org/oppia/android/testing/AssertionHelper.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.android.testing
+
+import org.junit.Assert.fail
+import kotlin.reflect.KClass
+import kotlin.reflect.full.cast
+
+object AssertionHelper {
```

Comment:
I suggest omitting the object entirely and just requiring that tests import the method directly. Also, suggest naming the file ``AssertionHelpers.kt`` since it may contain more than one in the future (otherwise we would just name it ``AssertThrows`` if we mean to only ever have this assertion function in it).

Reply:
Done.